src/py42/sdk/queries/fileevents/filters/exposure_filter.py

```diff
@@ -124,6 +124,14 @@ def choices():
         return get_attribute_keys_from_class(SyncDestination)
 
 
+class SyncDestinationUsername(FileEventFilterStringField):
+    """Class that filters events based on the name of the username associated with the cloud service
+    the file is synced with (applies to ``synced to cloud service`` events only).
```

Comment:
I don't see this listed in the UI under `EventType`. Is this one or what is this? Re: `synced to cloud service`

Reply:
I believe this refers to the value of `Exposure Type` in the UI. There's a value in the UI for "Synced to cloud service"